R/capacity/disasters-emergencies/england/fire-response-time.R

```diff
@@ -0,0 +1,123 @@
+response <- GET("https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/LAD21_FRA21_EW_LU/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json")
+content <- content(response, type = "application/json", simplifyVector = TRUE)
+
+fra_lad_lookup <- content$features$attributes |>
+  select(lad_code = LAD21CD, lad_name = LAD21NM, fra_code = FRA21CD, fra_name = FRA21NM) |>
+  filter(!str_detect(fra_name, "Wales"))
+
+# Check coverage of LADs in dataset ----
+# 2021 Local Authorities list
+# Source: https://geoportal.statistics.gov.uk/datasets/967a3660c4aa49819731ceefe4008d76_0/explore
+response <- GET("https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/LTLA21_UTLA21_EW_LU/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json")
+content <- content(response, type = "application/json", simplifyVector = TRUE)
+
+ltla_utla_lookup <- content$features$attributes |>
+  select(ltla_code = LTLA21CD, ltla_name = LTLA21NM, utla_code = UTLA21CD, utla_name = UTLA21NM)
+         
+ltla_utla_lookup |>
+  anti_join(fra_lad_lookup, by = c("ltla_code" = "lad_code")) |> 
+  filter(str_detect(ltla_code, "^E")) 
+
+fra_lad_lookup  |>
+  anti_join(ltla_utla_lookup, by = c("lad_code" = "ltla_code")) |> 
+  filter(str_detect(lad_code, "^E")) 
+
+# Check of how many LADs to a FRA ----
+fra_lad_lookup |>
+  group_by(fra_code) |>
+  summarise(count = n()) |>
+  summary()
+
+fra_lad_lookup |>
+  distinct(fra_name)
+# 47 FRAs
+
+
+# Check matches between datasets ----
+fra_lad_lookup |>
+  anti_join(fire_response, by = c("fra_name" = "frs")) |>
+  distinct(fra_name)
+
+fire_response |>
+  anti_join(fra_lad_lookup, by = c("frs" = "fra_name"))
+
+# Manual fixes of some names to allow matching
+# Isles Of Scilly had no dwelling fires for 2021/22 
+fire_response_updated <- fire_response |>
+  mutate(frs = case_when(
+    frs == "Devon and Somerset" ~ "Devon & Somerset",
+    frs == "Dorset and Wiltshire" ~ "Dorset & Wiltshire",
+    frs == "Greater London" ~ "London Fire and Emergency Planning Authority",
+    frs == "Nottinghamshire" ~ "Nottinghamshire and City of Nottingham",
+    frs == "Staffordshire" ~ "Stoke-on-Trent and Staffordshire",
+    frs == "Buckinghamshire" ~ "Buckinghamshire & Milton Keynes",
+    frs == "Berkshire" ~ "Royal Berkshire",
+    frs == "Durham" ~ "County Durham and Darlington",
+    frs == "Hereford and Worcester" ~ "Hereford & Worcester",
+    TRUE ~ frs
+  )) |>
+  bind_rows(list(frs = "Isles of Scilly", incidents = 0, station_response_time = NA))
+
+# Check matches after manual fixes
+fire_response_updated |>
+  anti_join(fra_lad_lookup, by = c("frs" = "fra_name"))
+
+fra_lad_lookup |>
+  anti_join(fire_response_updated, by = c("fra_name" = "frs")) |>
+  distinct(fra_name)
+
+# Join fire response time data to FRA to LAD lookup ----
+# Assumption: all LADs within an FRS have same response time as don't currently have data at a lower geography level
```

Comment:
I am not sure on the validity of this method. On the one hand, I think you could reason that some comparison between some areas adds nuance to the index, on the other, could it potentially wash out the effects of other indicators?

@matthewgthomas would be interested for your opinion on this too.

Reply:
Yeah that's fair. Not sure if you saw my comment in the PR that I'm currently in contact with Home Office to get Local Authority level data on response time to fires, they are just going through checks. But not sure if want to wait until have that data before uploading the D&E index onto the VCSEP toolkit.

Reply:
This is a decision for @matthewgthomas. I would probably exlclude the data.

Reply:
@matthewgthomas So it looks like 33% of the Local Authorities stay in the same decile if remove the fire response times and 42% move up or down one decile. Also included the spread of the rank changes. Happy to discuss. 
![image](https://user-images.githubusercontent.com/20212709/158802656-2b328079-b4d4-4595-84b5-f3e0088d87b4.png)
![image](https://user-images.githubusercontent.com/20212709/158803002-97fdef03-204d-4998-9a22-7145d7eb92ff.png)

Reply:
Interesting. I would say given the change isn't huge in this case, let's leave it in for completeness, and for the refined models we build down the line ensure we get stats with better coverage.

Reply:
Agree - happy to exclude.

Reply:
Removed in commit https://github.com/britishredcrosssociety/resilience-index/pull/10/commits/41d6ccdb3487ac9af09871180cd915042e4e4382.